ambassador/tests/kat/t_extauth.py

```diff
@@ -357,6 +369,9 @@ def queries(self):
         # [5]
         yield Query(self.url("target/"), headers={"X-Forwarded-Proto": "https"}, expected=200)
 
+        # [6]
+        yield Query(self.url("target/unauthed/"), headers={"Requested-Status": "200"}, expected=200)
```

Comment:
Thanks for adding this check. I think you want to make sure that Envoy will skip the authorization process. Would make more sense to set Requested-Status to HTTP 401, but expect an HTTP 200 response?

Reply:
Yep, I'll do that. Is there any documentation that explains what the provided headers do on the backend?

Reply:
Setting `Requested-Status` to 401 makes this test return a 401. I added more assertions to make sure we're not hitting the auth service. Is this doing what I think it is?

Reply:
Right, `requested-status` is actually telling to the backend what to do. The assertions that you added are perfect. 

There is a KAT [tutorial](https://github.com/datawire/kat/blob/master/docs/tutorial.rst), but nothing very specific about what the headers do to the backend. The quickest way to know is by looking at the backend server implementation, e.g. [requested-status](https://github.com/datawire/ambassador/blob/3cd648c1133aac76af3bc54ebe6bf8dfa1e541aa/kat/backend/server.go#L51).